eng/Versions.props

```diff
@@ -1,5 +1,5 @@
 <Project>
-  <PropertyGroup>
+  <PropertyGroup Label="Versioning">
     <RepositoryUrl>https://github.com/dotnet/dotnet-monitor</RepositoryUrl>
     <VersionPrefix>5.0.0</VersionPrefix>
```

Comment:
should we rev this?

Reply:
We're still developing the 5.0.0 version; what is getting changed for each release is the PreReleaseVersionIteration property. The release branch has it set to 4 whereas the main branch has it set to 5.